package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "env $(sed 's/ # .*//' .env) supervisor app",
-    "test": "npm run lint && jest test --runInBand",
+    "test": "yarn lint && mocha",
```

Comment:
Noticed this was still using `npm`, so updated it to use `yarn` for consistency...